app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    person_list = [Person(person.get("name"), person.get("age")) for person in people]
+
+    for person in people:
+        if person.get("wife"):
+            Person.people.get(
+                person["name"]
+            ).wife = Person.people.get(person["wife"])
```

Comment:
```suggestion
            wife = Person.people[person["wife"]]
            Person.people[person["name"]].wife = wife
```
Fix readability. The same with husband